internal/rollup/terser-wrapped.js

```diff
@@ -74,51 +111,99 @@ function runterser(inputFile, outputFile, sourceMapFile) {
     },
     'mangle': !debug,
   };
+  return terserConfig;
+}
 
-  let config = configFile;
-  if (!config) {
-    config = tmp.fileSync({keep: false, postfix: '.json'}).name;
+/**
+ *
+ * @param {{inputFile: string, outputFile: string, sourceMapFile: string}[]} files
+ * @param {boolean} debug
+ */
+async function runTerserSerial(files, debug) {
+  const Terser = require(TERSER_MODULE_LOCATION);
+  for (const f of files) {
+    // run one set of terser at a time
+    await runTerser(f.inputFile, f.outputFile, f.sourceMapFile, Terser, debug);
   }
+}
 
-  fs.writeFileSync(config, JSON.stringify(terserConfig));
-
-  const args = [
-    require.resolve('build_bazel_rules_nodejs_rollup_deps/node_modules/terser/bin/uglifyjs'),
-    inputFile, '--output', outputFile, '--config-file', config
-  ];
+/**
+ *
+ * @param {{inputFile: string, outputFile: string, sourceMapFile: string}[]} files
+ * @param {boolean} debug
+ */
+async function runTerserParallel(files, debug) {
+  // TODO: make this configurable
```

Comment:
Same here, would be a good candidate for config, but we'll worry about it later I think